Confus/Confus/MazeGenerator.cpp

```diff
@@ -160,6 +160,11 @@ namespace Confus
         }
     }
 
+    void MazeGenerator::resetMaze()
+    {
+
+    }
```

Comment:
Note that this could be implemented by storing the initial seed (which is currently hardcoded in the game class) and simply re-using that